miso-web/src/main/webapp/pages/listRuns.jsp

```diff
@@ -35,15 +35,29 @@
 
 <div id="maincontent">
   <div id="contentcolumn">
-    <h1>
-      <span id="totalCount">Runs
-      </span>
-    </h1>
+    <h1>Runs</h1>
+    <ul class="sddm">
+      <li>
+        <a onmouseover="mopen('addrunmenu')" onmouseout="mclosetime()">Add Run
+          <span style="float:right" class="ui-icon ui-icon-triangle-1-s"></span>
+        </a>
+        <div id="addrunmenu" onmouseover="mcancelclosetime()" onmouseout="mclosetime()" style="visibility: hidden;">
+          <c:forEach items="${platformTypes}" var="pt">
+            <a href="/miso/run/new/${pt.name()}">Add ${pt.key}</a>
+          </c:forEach>
+        </div>
+       </li>
+    </ul>
+
     <table class="display no-border" id="listingRunsTable">
     </table>
     
     <script type="text/javascript">
       jQuery(document).ready(function () {
+          Run.platformTypes = [
```

Comment:
This isn't necessary anymore. Anything that needs it can use `Constants.platformTypes` instead.